src/formatter.ts

```diff
@@ -44,9 +44,9 @@ import {
 } from "./grammar/ast";
 
 /**
- * PrettyPrinter class provides methods to format and indent Tact code.
+ * Formatter provides methods to format and indent Tact code.
  */
-class PrettyPrinter {
+export class Formatter {
```

Comment:
Hmmm, technically this is a printer, because formatter is a parser + printer. Perhaps we should rename the file?

Reply:
let's go with `prettyPrinter.ts`